package.json

```diff
@@ -25,21 +25,19 @@
   },
   "scripts": {
     "omega-edit-download": "node -e \"require('./build/scripts/omega_edit_download.ts').downloadServer()\"",
-    "vscode:prepublish": "yarn run package-ext",
     "precompile": "node -p \"'export const LIB_VERSION = ' + JSON.stringify(require('./package.json').version) + ';'\" > src/version.ts",
     "compile": "tsc -p ./ && yarn omega-edit-download",
     "lint": "yarn run prettier src -c",
     "watch": "yarn omega-edit-download && webpack --watch --devtool nosources-source-map --config ./build/extension.webpack.config.js",
     "watch2": "tsc -watch -p ./",
-    "package": "vsce package",
+		"webpack": "webpack --mode production --config ./build/extension.webpack.config.js",
+		"package": "vsce package",
     "publish": "vsce publish",
-    "package-ext": "webpack --mode production --config ./build/extension.webpack.config.js",
     "pretest": "yarn run compile",
     "test": "node node_modules/mocha/bin/_mocha -u tdd --timeout 999999 --colors ./out/tests",
     "sbt": "sbt universal:packageBin",
-    "prebuild": "node -e \"require('./build/scripts/process_build.ts').prebuild()\"",
-    "build": "yarn sbt && yarn install && yarn compile && yarn package",
-    "postbuild": "node -e \"require('./build/scripts/process_build.ts').postbuild()\""
+    "prebuild": "yarn sbt && yarn install && yarn compile && yarn webpack",
+    "build": "node -e \"require('./build/scripts/process_build.ts').build_package()\""
```

Comment:
Just out of curiosity, what's the intended difference between compile/precompile and build/prebuild? What kind of things should happen in one and what should happen in the other? The two seems like similar concepts and it's not clear to me what the difference is.

For example, yarn omega-edit-download happens in compile, but that's not actually compiling anything, right? It just downloads a zip? Seems like maybe that wants to happen in a build or prebuild stage? Likewise, "yarn sbt" does potentially compile scala code, so feels like it wants to happen in compile stage? Or maybe "sbt compile" should happen in compile stage, and "sbt universal:packageBin" should happen in the build stage?

I'm not sure it really matters, but it's not totally clear to me. Just trying my best to understand yarn.

Reply:
So basically compile should be everything dealing with creating the out js file, extension webpack file, I think technically it should have sbt running as well, maybe I left that out to make it quicker I am not fully sure. But I think with removing the `vsce package` command we actually replace the `build` script with `package` as that is what it is really doing anyway. Would that make sense to do?

Reply:
Also instead of the script being `build/scripts/process_build.ts` you think `build/scripts/package.ts` would be better?

Reply:
I removed the copying of node_modules as it seemed to break in some cases, so I copy in `yarn.lock` then run `yarn install`

Reply:
Renaming to package.ts seems reasonable to me. yarn.lock + yarn install seems fine too.